back/node_watcher/src/tasks/types.ts

```diff
@@ -123,13 +124,28 @@ export enum ProposalStatus {
   TABLED = 'Tabled',
 }
 
-export enum ReferendumStatus {
-  CANCELLED = ' Cancelled',
-  EXECUTED = 'Executed',
-  NOTPASSED = 'NotPassed',
-  PASSED = 'Passed',
-  STARTED = 'Started',
-}
+export const referendumStatus = {
```

Comment:
That's not a type.. is that ok to have this here?
Also because it's not a type, I can't use it above for instance in the `NomidotReferendum`, I have to use a string. 
Is there is a nicer way to do?